app/views/hyrax/datasets/show.html.erb

```diff
@@ -22,12 +22,18 @@
         <% end %>
       </div>
       <div class="contextual-info col-sm-3 text-center">
+        <% if @presenter.ancestor_trail.present? %>
+          <%= render 'vertical_breadcrumb', presenter: @presenter %>
+        <% end %>
         <%= render 'representative_media', presenter: @presenter, viewer: false unless @presenter.universal_viewer? %>
         <%= render 'altmetric_badge', presenter: @presenter %>
         <%# TODO: consider in future whether to add this partial back in %>
         <%#= render 'citations', presenter: @presenter %>
         <%= render 'social_media' %>
-        <%= render 'relationships', presenter: @presenter %>
+
+        <% if @presenter.grouped_presenters.present? %>
+          <%= render 'relationships', presenter: @presenter %>
+        <% end %>
```

Comment:
@seanaery I'm curious about this change.  It seems to have the effect of removing the Relationships section from top-level works but including it in non-top-level works.  Is that the intent?  I think we may want to rethink the Relationships section as a result of the work in this PR and I recognize that doing that is kind of outside the scope of this PR ... but I'm curious why you made this change in this PR.

Reply:
@coblej Good catch. Revising the PR. We can explore revisions to this section at a later date.